batcher/aligned-batcher/src/connection.rs

```diff
@@ -21,7 +21,9 @@ pub(crate) async fn send_batch_inclusion_data_responses(
     finalized_batch: Vec<BatchQueueEntry>,
     batch_merkle_tree: &MerkleTree<VerificationCommitmentBatch>,
 ) -> Result<(), BatcherError> {
-    for (vd_batch_idx, entry) in finalized_batch.iter().enumerate() {
+    // iter in reverse because each sender wants to receive responses in ascending nonce order
+    // and finalized_batch is ordered as the PriorityQueue , low max_nonce first && high nonce first.
```

Comment:
```suggestion
    // and finalized_batch is ordered as the PriorityQueue , low max_nonce first && high nonce last.
```

Reply:
PriorityQueue is ordered by which elements do we discard first.
So we first discard proofs of low max_fee and high nonce. Because we want the high max_fee and LOW nonce to go in first